tests/yast2_ncurse/yast2_dns_server.pm

```diff
@@ -0,0 +1,101 @@
+# SUSE's openQA tests
```

Comment:
we can create it under /console/ as we don't have any clear structure in mind, I think this just works because you had a typo `yast2_ncurse`, due to two test modules cannot be named the same even if they are in different folder in openQA. Therefore I'd recommend:
`console/yast2_dnsserver.pm`.